dependencies.gradle

```diff
@@ -15,6 +15,14 @@ dependencies {
 
     compile("curse.maven:journeymap-32274:2367915")
 
+    //compile("curse.maven:xaeros-minimap-263420:3502000")
+    //compile("curse.maven:xaeros-world-map-317780:3502011")
+    // need to use deobfuscated jars in dev env or it will crash on world load
+    // grab regular jars, run them through BON2, dump them in libs/, comment out maven gets above
+    // fucking annoying, why cant xaero just provide a dev jar
+    compile files("libs/xaeros-minimap-263420-3502000-deobf")
+    compile files("libs/xaeros-world-map-317780-3502011-deobf")
```

Comment:
Yeah, that's a bummer, you can move things into separate repo and setup CI with jetpackio, so you don't need to keep jars here. See https://gist.github.com/jitpack-io/f928a858aa5da08ad9d9662f982da983 for example

Reply:
ah, it's ARR and you need to do it yourself for each dev environment, nvm...

Reply:
That won't help with ARR :-/
Unless you create a project that takes the jar from course and produces itself + deobf

Reply:
even if you do you'll still be distributing original code, just that part of it will be deobf

Reply:
The files are not committed and listed on .gitignore. It's done correctly. you have to decompile and add them yourself after you cloned the repo